queries.css

```diff
@@ -232,6 +233,10 @@
     margin: 50px 0 0 80px;
   }
 
+  .about .about_details {
+    font-size: 20px;
```

Comment:
Increased font size in the About section of the desktop version for better readability.